app/assets/javascripts/member-facing.js.erb

```diff
@@ -19,6 +19,8 @@
 require("<%= Settings.external_js_path %>");
 <% end %>
 
+window.URI = require('urijs');
```

Comment:
+1 on using a library, parsing urls is not fun! 

I believe the issue is that this is used in `sou-assets/templates` so we can't `require('uri')` locally there. Unfortunately.